unity-client/Assets/ABConverter/Core.cs

```diff
@@ -160,13 +163,92 @@ void UpdateLoop()
                         OnFinish: (skippedAssetsCount) =>
                         {
                             this.skippedAssets = skippedAssetsCount;
+
+                            if(this.skippedAssets > 0)
+                                state.lastErrorCode = ErrorCodes.ASSET_BUNDLE_BUILD_FAIL;
```

Comment:
This is not entirely true as we might have converted most assets successfully. In the worst case, can we have an alternative error code for this case?

Reply:
Updated it with a new error code, tell me what you think